lib/core/__tests__/routing.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {docsRouting, blogRouting} = require('../routing');
+
+describe('Routing', () => {
```

Comment:
Have two describe blocks, one for docs and one for blog, and add an empty line between each `test` block.